DATA/production/calib/hmp-pedestals-processing.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+# ------------------------------------------------------------------------
+#         ALICE HMPID detector
+# Workflow to calculate pedestals   v.0.2  = 2/03/2022
+#
+#  Extra Env Variables:
+#     HMP_SIGMACUT : The value of sigams for the threshold cut [=4]
+#     HMP_CCDB_REC : True if we want the recording into ALICE CCDB
+#     HMP_PED_TAG :  A string that tags the pedestals [=Latest]
+#
+#
+#
+#   Auth. A.Franco  - INFN  Sez.BARI - ITALY
+# ------------------------------------------------------------------------
+
+source common/setenv.sh
+# env  > /home/fap/dump_env.txt
+
+# ---------------------------------------------------------------------------------------------------------------------
+# Set general arguments
```

Comment:
I would recommend to source `getCommonArgs.sh` instead of copy&pasting this part

Reply:
Done ! Thank you

Reply:
Did you test this? getCommonArgs is in the common folder as is setenv.sh, so I assume you have to do `source common/getCommonArgs.sh`.
Did you test this workflow at P2? You can check out a local clone with this PR to your home folder on the EPN, and then use the workflow configuration method = repository path pointing to your DATA path to test it, before merging to the official repository.

Reply:
yes, i did all the test in my DATA path before the PR ...